integrationExamples/videoModule/jwplayer/bidMarkedAsUsed.html

```diff
@@ -76,17 +76,15 @@
           console.log('player setup failed: ', e);
         });
 
-        pbjs.onEvent('videoPlaybackRequest', (e) => {
```

Comment:
why does this not work, even if the player does not exist yet? Is the problem that you may complete multiple auctions before the video player inits? If that's the case, is it really better to play old ads instead of discarding them and running a new auction for the next one?

Reply:
If you run an auction in the Header, the auction is likely to complete before the player is instantiated. The video module will attempt to load the ad into the player, but since the player is not instantiated, the ad is lost. 

I think if the publisher runs an auction in the header and configures the ad unit so that the ad is rendered by the Video Module, then he explicitly wants the ad to be queued. Otherwise he can listen to Video events to coordinate his auction.

Reply:
I'd think a `videoPlaybackRequest` event would have to come from the player, it's surprising to me that it could fire when the player is not ready.